guides/common/modules/con_content-view-environment-categories.adoc

```diff
@@ -0,0 +1,15 @@
+[id="content-view-environment-categories"]
+= Content view environment categories
+
+The content view environments available in your organization fall into the following categories:
+
+Library environment::
+The system assigns the Library environment to represent the Library lifecycle environment and the *Default Organization View* content view.
```

Comment:
```suggestion
{Project} assigns the Library environment to represent the Library lifecycle environment and the *Default Organization View* content view.
```
suggestion/question: clarify wording.

Is that true? If not, what do you mean by "system"?

Reply:
technical answer: Katello code tells Candlepin what the environment name should be :)

{Project} sounds good to me